pyfr/integrators/dual/pseudo/pseudocontrollers.py

```diff
@@ -63,6 +63,7 @@ def _update_pseudostepinfo(self, niters, resid):
 class DualNonePseudoController(BaseDualPseudoController):
     pseudo_controller_name = 'none'
     pseudo_controller_needs_lerrest = False
+    controller_has_variable_dt = False
```

Comment:
AttributeError when we try to validate the formulation here [https://github.com/PyFR/PyFR/blob/develop/pyfr/integrators/dual/pseudo/pseudocontrollers.py#L12](url)

Reply:
Okay. In that case I'd rewrite https://github.com/PyFR/PyFR/blob/e3e724607f9a0a5f74332c0db54ee35f0641d826/pyfr/solvers/euler/elements.py#L51 so that everything is inside an `if shock_capturing == 'entropy-filter':` block.  Then you can first check for dual time stepping and then check the time step is constant.  This will avoid the dodgy attribute access entirely.